packages/gatsby-source-shopify/README.md

```diff
@@ -248,79 +274,111 @@ module.exports = {
     {
       resolve: "gatsby-source-shopify",
       options: {
-        password: process.env.SHOPIFY_SHOP_PASSWORD,
-        storeUrl: process.env.GATSBY_SHOPIFY_STORE_URL,
+        password: process.env.SHOPIFY_APP_PASSWORD,
```

Comment:
Side note: we should add these updated env var names to the migration guide

Reply:
I don't believe we would need to. Because this code is in the gatsby-config.js file, they can keep their current .env variables exactly the same and the repo will still work as intended. Changing the names of the variables just made it more clear where they are coming from by using the same terminology Shopify does.